ibis/tests/expr/test_rules.py

```diff
@@ -290,6 +290,21 @@ def test_invalid_column_or_scalar(validator, value, expected):
         validator(value)
 
 
+@pytest.mark.parametrize(
+    ('check_table', 'value', 'expected'),
+    [
+        (table, "int_col", table.int_col),
+        (table, table.int_col, table.int_col),
+    ],
+)
+def test_valid_column_from(check_table, value, expected):
+    class Test:
+        table = check_table
+
+    validator = rlz.column_from("table")
+    assert validator(value, this=Test()).equals(expected)
+
```

Comment:
any check for the raising?, what about the cases where the column doesn't have a name?

Reply:
With a column that doesn't have a name you'd raise `IbisTypeError`. I can add invalid tests.